javascript-localization.pug

```diff
@@ -329,4 +342,30 @@ block content
     window.disableCookiesHook = function () {
       $('.yt-video').hide();
       $('.watch-yt-video-btn').show();
-    };
\ No newline at end of file
+    };
+
+  script(type="application/ld+json").
```

Comment:
What's this?

Reply:
Google structured data https://developers.google.com/search/docs/guides/intro-structured-data?hl=de